WalletWasabi/Http/Models/HeaderSection.cs

```diff
@@ -51,7 +52,7 @@ public static HeaderSection CreateNew(string headersString)
 					{
 						break;
 					}
-					hs.Fields.Add(HeaderField.CreateNew(field));
+					hs.Fields.Add(await HeaderField.CreateNewAsync(field));
```

Comment:
It would be good to have this in two different lines.

Reply:
I removed the streamreaders, hence the source of the asynchronoucity in status lines.